src/os-linux.h

```diff
@@ -2,14 +2,16 @@
 #define _GNU_SOURCE
 #include <netinet/in.h>
 #include <netinet/ip.h>
-#include <netinet/igmp.h>
```

Comment:
What is the reason that you do not want to use de-facto standard netinet/igmp.h header file?